packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -357,7 +359,8 @@ export class DeltaManager
     }
 
     public shouldJoinWrite(): boolean {
-        return this.clientSequenceNumber !== this.clientSequenceNumberObserved;
+        // We don't have to wait for ack for topmost NoOps. So subtract those.
+        return (this.clientSequenceNumber - this.lastNoOps) !== this.clientSequenceNumberObserved;
```

Comment:
!== will not work here. Consider two cases:
- noops were sent but not received at all by this client (for any reason)
- noops were sent, but some of them were acked (sequenced and received back).

These two cases will produce same values for thisclientSequenceNumber & lastNoOps, but different for this.clientSequenceNumberObserved.
Thus it should be <=

Reply:
Yes, should be "<".